pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
@thockin I've been looking at the pattern gce_pd uses for this; it tries to unmount the directory if it's already a mountpoint, and remount it.  The `isMountPoint` function is in the gce_pd plugin; the build flags are a little weird; gce_pd uses `!windows` as the flag for the code with unix-mount awareness; the `pkg/util/mount` package uses `linux` as the build flag.  I think this code is closely related enough that it should probably all just go into `pkg/util/mount`, but we need to reconcile the build flags first.

All that said - should we try to unmount an existing mountpoint at the tmpfs storage root?  If we make the tmpfs root purely a function of the root dir (ie, don't allow you to specify some arbitrary mountpoint for tmpfs), it should be fine to remount.  If we allow you to specify an arbitrary path, it might be dangerous to do this.

Personally I think we should:

1.  Make the tmpfs root a function of the root dir for the kubelet
2.  Unmount / remount an existing tmpfs mount.

Thoughts?